core/src/CMakeLists.txt

```diff
@@ -65,13 +61,19 @@ set(_srcs
 
 set(_private_headers
   util/usFrameworkPrivate.h
+  util/usLDAPExpr_p.h
+  util/usProperties_p.h
   util/usThreads_p.h
   util/usUtils_p.h
   util/usWaitCondition_p.h
 
   service/usServiceHooks_p.h
+  service/usServiceListenerEntry_p.h
```

Comment:
Is this change related to LDAP issue ? if not i'd suggest creating another branch for this issue. I believe this falls under issue#67